MADSkillsNavigationSample/app/build.gradle

```diff
@@ -21,7 +21,7 @@ apply plugin: "androidx.navigation.safeargs.kotlin"
 apply plugin: 'kotlin-kapt'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 31
```

Comment:
Nit: you can use the versions in `versions.gradle`

Reply:
We don't own this sample and it doesn't build so my plan is to deprecate it, so I didn't want to change too much.